docs/examples/pydantic_ai.md

```diff
@@ -0,0 +1,45 @@
+[Pydantic AI](https://ai.pydantic.dev/) is a Python agent framework built by the Pydantic team that uses Pydantic validation for [structured output](https://ai.pydantic.dev/output/#structured-output) schema generation and validation.
+By specifying an `output_type` on an Agent, you can constrain the LLM to return data that matches your Pydantic model schema.
+
+## LLM Structured Output
+
+=== "Simple"
```

Comment:
I'd say keeping one example is fine with a validator. Let's also make use [the validation context](https://ai.pydantic.dev/output/#validation-context) to make the example more lifelike, by passing in a small list of valid countries.

Reply:
sorry for taking so long, modified the example to use validation context and kept it to one exmaple now

Reply:
@dsfaccini can you pleas look at the CI issue?